README.md

```diff
@@ -31,18 +36,15 @@ JCodec is free software distributed under FreeBSD License.
 
 * Video
     * Improve H.264 encoder: CABAC, rate control;
-    * Performance optimize H.264 decoder;
-    * Implement H.264 encoder on RenderScript;
```

Comment:
RenderScript was a good idea

Reply:
Agree, but realistically nobody is going to do it in the next six months. I am don't want to set the expectations high.